src/sweeper/sweepers/metadata.py

```diff
@@ -6,14 +6,15 @@
 '''
 import re
 from os.path import dirname, join, realpath
+from .. import credentials
 
 from arcpy import Exists
 from arcpy import metadata as md
 from bs4 import BeautifulSoup
 
 #: these constants were copied from https://github.com/agrc/agol-validator/blob/master/validate.py
 #: Tags or words that should be uppercased, saved as lower to check against
-UPPERCASED_TAGS = ['2g', '3g', '4g', 'agol', 'agrc', 'aog', 'at&t', 'blm', 'brat', 'caf', 'cdl', 'daq', 'dfcm', 'dfirm', 'dwq', 'e911', 'ems', 'fae', 'fcc', 'fema', 'gcdb', 'gis', 'gnis', 'hava', 'huc', 'lir', 'lrs', 'lte', 'luca', 'mrrc', 'nca', 'ng911', 'nox', 'npsbn', 'ntia', 'nwi', 'plss', 'pm10', 'psap', 'sao', 'sbdc', 'sbi', 'sgid', 'sitla', 'sligp', 'trax', 'uca', 'udot', 'ugs', 'uhp', 'uic', 'us', 'usao', 'usdw', 'usfs', 'usfws', 'usps', 'ustc', 'ut', 'uta', 'vcp', 'vista', 'voc']
+UPPERCASED_TAGS = ['2g', '3g', '4g', 'agol', 'agrc', 'aog', 'at&t', 'blm', 'brat', 'caf', 'cdl', 'daq', 'dfcm', 'dfirm', 'dnr', 'dsl', 'dwq', 'e911', 'ems', 'fae', 'fcc', 'fema', 'gcdb', 'gis', 'gnis', 'hava', 'huc', 'lir', 'lrs', 'lte', 'luca', 'mrrc', 'nca', 'ng911', 'nox', 'npsbn', 'ntia', 'nwi', 'nws', 'plss', 'pm10', 'psap', 'sao', 'sbdc', 'sbi', 'sgid', 'sitla', 'sligp', 'trax', 'uca', 'udot', 'ugrc', 'ugs', 'uhp', 'uic', 'us', 'usao', 'usdw', 'usfs', 'usfws', 'usps', 'ustc', 'ut', 'uta', 'vcp', 'vista', 'voc']
```

Comment:
**suggestion**: it would be a good idea to put these values into a file type that doesn't require the module to be installed to update so modifying this list can be done simply.

Reply:
Are you thinking like a text file or something that python reads in and builds the list from? 

Another aspect is that it would be good to figure out a way to sync this list between Sweeper and Auditor so that we only need to make changes in one place. Let me think on that and make it a future enhancement for Sweeper/Auditor so as not to hold this up (maybe a dedicated repo just for that list that we can then pull independently of the client programs, and reference the file location in `credentials.py`?)

Reply:
Ok, I didn't realize the two project shared this. Think about it and let me know what options come to mind.

Reply:
And now that I read the comment above the list, I'm realizing it still references Auditor's old repo name and link.

Reply:
I created an issue for this.